lib/src/core/core.dart

```diff
@@ -1,25 +1,25 @@
 library bond_core;
 
-import 'package:bond_core/src/form/form_state.dart';
+import '../../src/form/form_state.dart';
```

Comment:
don't use relative path

and remove all import and LoginFormStateNotifier from file